src/TypedPetri.jl

```diff
@@ -208,17 +208,19 @@ end
 
 Assumes a single species type and a single transition type.
 """
-function pairwise_id_typed_petri(type_net, stype, ttype, args...)
+function pairwise_id_typed_petri(type_net, stype, ttype, args...;
+                                 codom_net=nothing)
```

Comment:
This keyword parameter is a workaround. It should be removed and fixed later.

Reply:
@epatters could you put a todo comment note on the line just so we make sure to remember?

Reply:
I went ahead and put a todo note here just heads up @epatters. Just saw that you marked this as a draft so I wanted to give you a heads up since it might have merge conflicts on your end. My bad!

Reply:
Thanks Micah!